python/tests/test_petab.py

```diff
@@ -0,0 +1,60 @@
+from petab.C import *
+import parpe.petab
+import pandas as pd
+
+
+def test_get_parameter_override_id_to_placeholder_id():
+    """Test get_parameter_override_id_to_placeholder_id"""
+
+    observable_df = pd.DataFrame(data={
+        OBSERVABLE_ID: ['obs1', 'obs2'],
+        OBSERVABLE_FORMULA: ['1.0',
+                             'observableParameter1_obs2 + '
+                             'observableParameter2_obs2'],
+        NOISE_FORMULA: ['noiseParameter1_obs1', '1.0']
+    })
+    observable_df.set_index(OBSERVABLE_ID, inplace=True)
+
+    measurement_df = pd.DataFrame(data={
+        OBSERVABLE_ID: ['obs1', 'obs1', 'obs2', 'obs2'],
+        OBSERVABLE_PARAMETERS: ['', '', '1.0;1.0', 'override1;override2'],
+        NOISE_PARAMETERS: ['1.0', 'override3', '', '']
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['NOISE_PARAMETERS' may be undefined, or defined from star imports: petab.C](https://app.codacy.com/manual/dweindl/parPE/pullRequest?prid=4922781)